Dockerfiles/agent/README.md

```diff
@@ -76,16 +76,22 @@ DD_AC_INCLUDE = "image:cp-kafka image:k8szk"
 
 Please note that the `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.
 
+### Datadog Cluster Agent
+
+As with Agent v6.2, you can use the [Datacong Cluster Agent](#https://github.com/DataDog/datadog-agent/blob/master/docs/cluster-agent/README.md).
+
+Cluster level features are now handled by the cluster agent, and you will find a `[DCA]` notation next to the affected features. Please refer to the below user documentation as well as the technical documentation here for further details on the instrumentation.
```

Comment:
Whats the experience if the cluster agent isn't deployed? Does it fall back to legacy behavior?

Reply:
Yes, the RBACs will need to be set accordingly though. Similarly, if the cluster agent is configured but not reachable we fallback to the previous behavior (which also requires legacy RBAC setup)